src/Symfony/Bridge/Twig/Form/TwigRendererEngine.php

```diff
@@ -154,7 +154,7 @@ protected function loadResourcesFromTheme($cacheKey, &$theme)
     {
         if (!$theme instanceof Template) {
             /* @var Template $theme */
-            $theme = $this->environment->loadTemplate($theme);
+            $theme = $this->environment->load($theme)->unwrap();
```

Comment:
@fabpot the reason why we need to unwrap here is that we rely on the internal `getBlocks` function of the Template class. We don't only want to use the block names, so the non-internal `getBlockNames` is not enough for us (we are also relying on the internal `displayBlock` signature which has more arguments)